src/page/MyShopPage/index.tsx

```diff
@@ -10,9 +11,16 @@ import EditShopInfoModal from './components/EditShopInfoModal';
 
 export default function MyShopPage() {
   const { isMobile } = useMediaQuery();
+  const location = useLocation();
   const {
     shopData, menusData, refetchShopData, isLoading,
   } = useMyShop();
+  const { resetAddMenuStore } = useAddMenuStore();
+  useEffect(() => {
+    if (location.pathname === '/') {
+      resetAddMenuStore();
+    }
+  }, [location, resetAddMenuStore]);
```

Comment:
```suggestion
  useEffect(() => {
      resetAddMenuStore();
  }, [location, resetAddMenuStore]);
```
`MyShopPage`의 `location.pathname`이 `/`로 고정되어 있기 때문에 조건문을 쓰지 않아도 될 것 같아요!

Reply:
그러네요 감사합니다 😊